src/app/pages/receive/receive-stx.tsx

```diff
@@ -16,7 +16,7 @@ export function ReceiveStxModal() {
 
   function copyToClipboard() {
     void analytics.track('copy_stx_address_to_clipboard');
```

Comment:
A bit less exciting but OK 😄

Reply:
haha, this was to trigger the build, making changes in `src` prior to other fixes. 

Also, though, I'm also of the opinion that exclamation marks shouldn't often be used in interfaces.

Reply:
😄 agreed. I read it as shouting at the users